docs/google.md

```diff
@@ -18,7 +18,7 @@ Heapster metrics are now being exported to Google Cloud Monitoring as custom met
 ### Metrics Dashboard
 To access the Google Cloud Monitoring dashboard go to: [https://app.google.stackdriver.com/](https://app.google.stackdriver.com/). Create a new dashboard and add the desired charts. Select the *Custom Metric* Resource Type and all Heapster metrics are under the `kubernetes.io` namespace. You can narrow down the query by the metric labels provided.
 
-It is also possible to query the Google Cloud Monitoring data directly using their [custom metric read API](https://cloud.google.com/monitoring/v2beta2/timeseries/list).
+It is also possible to query the Google Cloud Monitoring data directly using their [custom metric read API](https://cloud.google.com/monitoring/custom-metrics).
```

Comment:
Please point to the specific section with reference to API documentation: https://cloud.google.com/monitoring/custom-metrics/reading-metrics